.circleci/config.yml

```diff
@@ -4,11 +4,11 @@ executors:
   golang:
     docker:
       - image: circleci/golang:1.13
-    working_directory: /go/src/github.com/cosmwasm/wasmd
+    working_directory: /go/src/github.com/cosmwasm/wasmd-demo
```

Comment:
Why the rename here? This is for all tests

Reply:
😕 this was not intended.  should have marked this as draft.